starlark/interp.go

```diff
@@ -561,5 +563,8 @@ loop:
 			err = fr.errorf(f.Position(savedpc), "%s", err.Error())
 		}
 	}
+
+	fr.locals = nil
```

Comment:
This would be a bit more readable in a `defer` after the original assignment. Is there a significant overhead for  that?

Reply:
Yes, which is why this function has exactly one return statement. At some point I should add a static check.